python/taichi/examples/simulation/pic88.py

```diff
@@ -0,0 +1,93 @@
+# Authored by Luhuai Jiao
+# This is a 1D simulation of "two-stream instability" in Plasma Physicis.
+# Some settings of grids and particles are from "Introduction to Computational Plasma Physics"(《计算等离子体物理导论》，谢华生著)

```

Comment:
We should avoid use mixed English and Chinese here, you can use the ISBN number instead.

Reply:
Thanks for your advice! There is no Chinese in the new file now.